avocado/core/test.py

```diff
@@ -314,13 +314,6 @@ def _update_time_elapsed(self, current_time=None):
             current_time = time.time()
         self.time_elapsed = current_time - self.time_start
 
-    def report_state(self):
```

Comment:
this will break `sleeptenmin.py` test and probably the `GDBSubProcess` class from `avocado/utils/process.py`